### ISSUE TITLE
google-api-client warning messages when provisioning multiple instances

### ISSUE THREAD

**Issue by Temikus** [mitchellh/vagrant-google#54]
When provisioning multiple instances, we get warning messages from Google-api-client.

Reproduction:
1) Run `bundle exec vagrant up` with Vagrantfile.multiple_machines from examples.
2) Receive messages when machines are brought up:

```
/home/temikus/.gem/ruby/gems/google-api-client-0.8.6/lib/google/api_client/discovery/schema.rb:110: warning: already initialized constant Google::APIClient::Schema::Compute::V1::PathMatcher
/home/temikus/.gem/ruby/gems/google-api-client-0.8.6/lib/google/api_client/discovery/schema.rb:110: warning: previous definition of PathMatcher was here
/home/temikus/.gem/ruby/gems/google-api-client-0.8.6/lib/google/api_client/discovery/schema.rb:110: warning: already initialized constant Google::APIClient::Schema::Compute::V1::UrlMapTest
/home/temikus/.gem/ruby/gems/google-api-client-0.8.6/lib/google/api_client/discovery/schema.rb:110: warning: previous definition of UrlMapTest was here
```

Those messages may not be erroneous but may confuse the user, so need to be investigated.

**Comment by Temikus** [comment]
Note: Needs to be re-checked with gem installation. May be a byproduct of development mode execution.

**Comment by Temikus** [comment]
No longer reproducible since 0.1.5, so closing.

**Comment by KjetilM** [comment]
This has resurfaced on my setup .. 

See a lot of these. 

/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: already initialized constant Google::APIClient::Schema::Compute::V1::AttachedDisk
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: previous definition of AttachedDisk was here
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: already initialized constant Google::APIClient::Schema::Compute::V1::AttachedDiskInitializeParams
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: previous definition of AttachedDiskInitializeParams was here
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: already initialized constant Google::APIClient::Schema::Compute::V1::Autoscaler
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: previous definition of Autoscaler was here
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: already initialized constant Google::APIClient::Schema::Compute::V1::AutoscalerList
/.vagrant.d/gems/2.2.5/gems/google-api-client-0.8.7/lib/google/api_client/discovery/schema.rb:110: warning: previous definition of AutoscalerList was here

Etc. etc. 
Did anyone figure out the source? 
I've tried to expunge vagrant plugins just to make sure everything is fresh.

**Comment by KjetilM** [comment]
And it seems to be related to cleanup of values if you do something stupid at first. 
I had an error in the instance_group setting at first. so i commented that out. 
After that these errors came each time I did vagrant up. When i saw my mistake (used uppcase letter in name of instance_group as it only supports lowercase) then the error went away. 

But now if i comment the setting out, the messages return.

**Comment by Temikus** [comment]
@KjetilM Can you provide a Vagrantfile that you're able to reproduce this on?
(OFC obfuscate any API keys and PII)

**Comment by KjetilM** [comment]
`unless Vagrant.has_plugin?("vagrant-google")
  raise "vagrant-ohai plugin is not installed! Install with 'vagrant plugin install vagrant-google'"
end

unless Vagrant.has_plugin?("vagrant-ohai")
  raise "vagrant-ohai plugin is not installed! Install with 'vagrant plugin install vagrant-ohai'"
end

NODE_SCRIPT = <<EOF.freeze
  echo "Preparing node..."
  # ensure the time is up to date
  yum -y update
  yum -y install ntp
  systemctl start ntpd
  systemctl enable ntpd
  extip=$(curl -s http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip -H "Metadata-Flavor: Google")
  echo "== EXTERNAL IP: ${extip}"
EOF



#google Account settings
*REMOVED*

#set up box
Vagrant.configure("2") do |config|
  config.vm.box = "google/gce"
  config.vm.provision :shell, inline: NODE_SCRIPT.dup
  #google.instance_group = "chefdev"

  config.vm.define :web1 do |web1|
    web1.vm.provider :google do |google, override|
      google.google_project_id = $GOOGLE_PROJECT_ID
      google.google_client_email = $GOOGLE_CLIENT_EMAIL
      google.google_json_key_location = $GOOGLE_JSON_KEY_LOCATION
      google.machine_type = "g1-small"
      #google.instance_group = "chefdev"
      google.tags = ['http-server','apache']
      #google.disk_size = "5"
      #google.disk_type "pd-standard"
      google.image = "centos-7-v20170124"
      google.zone = "europe-west1-b"
      google.name = "web1"
      override.ssh.username = $LOCAL_USER
      override.ssh.private_key_path = $LOCAL_SSH_KEY
    end
  end

  config.vm.define :web2 do |web2|
    web2.vm.provider :google do |google, override|
      google.google_project_id = $GOOGLE_PROJECT_ID
      google.google_client_email = $GOOGLE_CLIENT_EMAIL
      google.google_json_key_location = $GOOGLE_JSON_KEY_LOCATION
      google.machine_type = "g1-small"
      #google.instance_group = "chefdev"
      google.tags = ['http-server','apache']
      #google.disk_size = "5"
      google.image = "centos-7-v20170124"
      google.zone = "europe-west1-b"
      google.name = "web2"
      override.ssh.username = $LOCAL_USER
      override.ssh.private_key_path = $LOCAL_SSH_KEY
    end
  end

  config.vm.define :lb1 do |lb1|
    lb1.vm.provider :google do |google, override|
      google.google_project_id = $GOOGLE_PROJECT_ID
      google.google_client_email = $GOOGLE_CLIENT_EMAIL
      google.google_json_key_location = $GOOGLE_JSON_KEY_LOCATION
      google.machine_type = "g1-small"
      #google.instance_group = "chefdev"
      google.tags = ['http-server','load-balancer']
      google.image = "centos-7-v20170124"
      google.zone = "europe-west1-b"
      google.name = "lb1"
      override.ssh.username = $LOCAL_USER
      override.ssh.private_key_path = $LOCAL_SSH_KEY
    end
  end    
end

`

**Comment by Temikus** [comment]
This should be fixed by #180 
Please, reopen if this resurfaces.
